conans/client/graph/graph_builder.py

```diff
@@ -328,7 +328,8 @@ def _conflicting_references(previous, new_ref, consumer_ref=None):
                         "in your root package."
                         % (consumer_ref, consumer_ref, new_ref, next(iter(previous.dependants)).src,
                            previous.ref, new_ref.name))
-            return True
+            return "Unresolvable conflict between {} and {}".format(previous.ref, new_ref)
```

Comment:
There are two more returns in this function. Should the be changed as well? Also, I believe that the one on line 341 was the actual return path in my case.

Reply:
The use case you provided is actually satisfied with this change, other cases might need other tests then.

Reply:
I guess the other paths are those parts that resulted in the test errors from my fix above. It still feels a bit icky to wrap a True into an exception, but since we're on the way to 2.0 anyways...